packages/commonwealth/client/scripts/views/components/component_kit/cw_avatar.tsx

```diff
@@ -12,6 +15,18 @@ type BaseAvatarProps = {
 
 type AvatarProps = BaseAvatarProps & { avatarUrl: string };
 
+export function getAvatarFromProfile(minimumProfile: MinimumProfile, size: number) {
```

Comment:
Is this the right place for these helper functions? Generally component kit should just be the components and not these rendering utilities.

Reply:
These helper functions needed to be moved because the NewProfilesController imported MinimumProfiles which had used this code. This code imports the mithrilInteropLayer which causes the tests to break.

Reply:
Can you find a third place to put them that's outside the component kit?